Firmware/variants/1_75mm_MK2-RAMBo10a-E3Dv6full.h

```diff
@@ -105,7 +105,9 @@ EXTRUDER SETTINGS
 #define HEATER_0_MINTEMP 15
 #define HEATER_1_MINTEMP 5
 #define HEATER_2_MINTEMP 5
+#define HEATER_MINTEMP_DELAY 15000                // [ms] ! if changed, check maximal allowed value @ ShortTimer
```

Comment:
Místo toho komentáře by mohlo být rozumnější někam dát konstrukci ve stylu

#include <limits.h>
static_assert(HEATER_MINTEMP_DELAY <= USHRT_MAX, "To je moc pro ShortTimer.");